scripts/shared/clusters.sh

```diff
@@ -2,15 +2,32 @@
 
 ## Kubernetes version mapping, as supported by kind ##
 # See the release notes of the kind version in use
-DEFAULT_K8S_VERSION=1.23
-declare -A kind_k8s_versions
+DEFAULT_K8S_VERSION=1.24
+declare -A kind_k8s_versions kind_kind_binaries
```

Comment:
```suggestion
declare -A kind_k8s_versions kind_binary
```

Clearer like this and less repetitive